02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+
+## Post Survey Processing {#overview-post}
+
+After data collection, there are a variety of activities that need to be conducted before the survey can be analyzed. Multiple decisions made during this post survey phase can assist researchers in reducing different error sources such as through weighting to account for the sample selection. Knowing the decisions researchers made in creating the final analytic data can impact how analysts use the data and interpret the results.
+
+
+### Data Cleaning and Imputation {#overview-post-cleaning}
+
+Post survey cleaning and imputation is one of the first steps researchers will do to get the survey responses into a dataset for use by analysts. Data cleaning can comprise of cleaning inconsistent data (e.g., with skip pattern errors or multiple questions throughout the survey being consistent with each other), editing numeric entry or open-ended responses for grammar and consistency, or recoding open-ended questions into categories for analysis. Each project should create their own rules for how to handle different cleaning situations, and there are no specific rules that must be followed. Instead, researchers should use their best judgement in ensuring data integrity remains and all decisions should be documented and available to those using the data in analysis. Each decision a researcher makes has an impact on *processing error*, so often researchers often will have multiple people review these rules or recode open-ended data and adjudicate any differences in an attempt to reduce this error.
+
+Another crucial step in post survey processing is imputation. Often times there is item nonresponse where respondents do not answer specific questions. If the questions are crucial to analysis efforts or to the research question, researchers will implement imputation in an effort to reduce *item nonresponse error*. However, imputation itself can introduce *processing error* as well, so researchers should consider the overall implications of imputing data compared to having item nonresponse. There are multiple ways that imputation can be conducted, we recommend reviewing other resources like {cite Dever}<!--Stephanie is this the right book or is there a better one on imputation?--> for more information.
```

Comment:
Not the Dever book. 

- Kim & Shao, Statistical Methods for Handling Incomplete Data
- Will think of more later.